.github/workflows/code-rabbit.yaml

```
@@ -0,0 +1,39 @@
+    steps:
+      - uses: coderabbitai/ai-pr-reviewer@latest
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+        with:
+          debug: false
+          review_simple_changes: false
+          review_comment_lgtm: false
+          language: ja-JP
+          summarize: |
+          次のmarkdownフォーマット形式を使用して、本PRの要約を出力します。
```

actionlint (1.7.9)

could not parse as YAML: could not find expected ':'

(syntax-check)